pkg/api/validation/validation_test.go

```diff
@@ -874,6 +911,11 @@ func TestValidatePersistentVolumeClaimUpdate(t *testing.T) {
 			oldClaim:          validUpdateClaim,
 			newClaim:          invalidUpdateClaimAccessModes,
 		},
+		"invalid-update-change-storage-class-annotation-after-creation": {
```

Comment:
Please add a test that updates another (=allowed) annotation - such update should be valid.